test/integration/webhookgithub_test.go

```diff
@@ -299,6 +300,11 @@ func postFile(client restclient.HTTPClient, event, filename, url string, expStat
 	if resp.StatusCode != expStatusCode {
 		t.Errorf("Wrong response code, expecting %d, got %d: %s!", expStatusCode, resp.StatusCode, string(body))
 	}
+	var newBuild buildapi.Build
+	if err = runtime.DecodeInto(kapi.Codecs.UniversalDecoder(), body, &newBuild); err != nil {
```

Comment:
I've added this, but I'm not sure if you meant to use the unmarshalled object or just use it for a sanity check?